src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1487,10 +1487,19 @@ public void TestUrldecode()
         [Test]
         public void TestDefault()
         {
-            Hash assigns = Hash.FromAnonymousObject(new { var1 = "foo", var2 = "bar" });
+            Hash assigns = Hash.FromAnonymousObject(new { var1 = "foo", var2 = "bar", array = new []{"1", "2"}, empty_array = new string[]{} });
             Helper.AssertTemplateResult("foo", "{{ var1 | default: 'foobar' }}", assigns);
             Helper.AssertTemplateResult("bar", "{{ var2 | default: 'foobar' }}", assigns);
             Helper.AssertTemplateResult("foobar", "{{ unknownvariable | default: 'foobar' }}", assigns);
+            Helper.AssertTemplateResult("foobar", "{{ false | default: 'foobar' }}", assigns);

```

Comment:
might want to use an explicit variable to be clear this is actually a boolean and not an unknown variable named 'false'